src/main/java/doob/model/Level.java

```diff
@@ -184,22 +184,9 @@ private void updateSurvival() {
     public void spawnBalls(long seed) {
 
         Random generator = new Random(seed);
-        int i = generator.nextInt() % 5;
-        /*switch(i) {
-            case 0: spawnSameSizeBalls(6, 16);
-                break;
-            case 1: spawnSameSizeBalls(4, 32);
-                break;
-            case 2: spawnSameSizeBalls(3, 32);
-                break;
-            case 3: spawnSameSizeBalls(2, 64);
-                break;
-            case 4: spawnSameSizeBalls(1, 128);
-                break;
-            default:
-                break;
-        }*/
-        spawnSameSizeBalls(6 - i, (int) (8 * Math.pow(2, i)));
+        int i = Math.abs(generator.nextInt() % 4);
```

Comment:
I mean this line. Isn't there a better way to use a random generator than to use Math.abs()?